samples/subsys/ipc/rpmsg_service/README.rst

```diff
@@ -128,3 +128,35 @@ remote) will appear on the corresponding serial ports:
    ...
    Remote core received a message: 98
    RPMsg Service demo ended.
+
+Building the application for stm32h747i_disco/stm32h7xx
+*******************************************************
+
+Using west, build the sample for both cores separately like so:
+
+.. code-block:: console
+
+    west build -d build-m7/ -b stm32h747i_disco/stm32h747xx/m7 .
+    west build -d build-m4/ -b stm32h747i_disco/stm32h747xx/m4 remote/
+
+Then, you can flash both cores of the disco board like so:
+
+.. code-block:: console
+
+    west flash -d build-m7/
+    west flash -d build-m4/
```

Comment:
does not use the correct doc template for building

Reply:
fixed